int-set/src/lib.rs

```diff
@@ -1593,11 +1593,11 @@ mod test {
         assert!(!set.contains(GlyphId::new(5)));
         assert!(set.contains(GlyphId::new(6)));
 
-        let Some(max) = set.iter().max() else {
-            panic!("should have a max");
-        };
+        // let Some(max) = set.iter().max() else {
+        //     panic!("should have a max");
+        // };
 
-        assert_eq!(max, GlyphId::new(u16::MAX));
+        // assert_eq!(max, GlyphId::new(u32::MAX as _));
```

Comment:
This test now takes a substantial amount of time with 32-bit glyph ids...

Reply:
I think we'll want to implement Domain for the glyphid 16 type, use that here in this test, and then add a new test that checks on the new GlyphId type but doesn't do max()

Reply:
Good idea. Made it so.